### DISCUSSION TITLE
How to get credentials for EcoTrend website

### DISCUSSION THREAD

**Discussion by BenRoe** [Ludy87/ecotrend-ista#136] [Q&A] [answered]
Hello, 
How can I register on the Ecotrend website.
Do i have to contact Ista, or my Landlord (Hausverwaltung)?

**Comment by Ludy87** [comment]
You must contact ista if the billing is resolved via ista.

**Reply by BenRoe** [reply, marked as answer]
Billing is from Ista.
Contacted ista, but they wrote that i have to contact my landlord (Hausverwaltung).
Waiting for the reply from landlord.
